doc/dev/coding-guidelines-and-review.txt

```diff
@@ -76,3 +76,34 @@ ready for review::
 **Note** that after you submit the comment you can check and uncheck
 the individual boxes on the formatted comment; no need to put x or y
 in the middle.
+
+CPython Checklist
+-----------------
+
+Here's a checklist for new CPython types::
+
+   - [ ] the CPython object name is of the form `khmer_${OBJECTNAME}_Object`
+   - [ ] Named struct with `PyObject_HEAD` macro
+   - [ ] `static PyTypeObject khmer_${OBJECTNAME}_Type` with the following
+     entries
+      - [ ] `PyObject_HEAD_INIT(NULL)`
```

Comment:
We can use ```PyVarObject_HEAD_INIT(NULL, 0)``` and help on the Python 2/3 compat.

Reference: http://python3porting.com/cextensions.html#object-initialization